src/testdir/test_i_ctrl_rsb.vim

```diff
@@ -0,0 +1,24 @@
+" Test for InsertCharPre autocmd
+
+def Test_i_ctrl_rsb_not_trigger_insertcharpre()
+    new
+    autocmd InsertCharPre <buffer> v:char = "a"
+    exe "normal! i\<C-]>"
+    call assert_equal('', getline(1))
+    bwipeout!
```

Comment:
Also need to delete the autocommand

Reply:
Technically not needed, as Vim automatically deletes a buffer-local autocmd when the buffer is deleted(wiped).